packages/snaps-controllers/src/snaps/registry/json.ts

```diff
@@ -226,7 +226,11 @@ export class JsonSnapsRegistry extends BaseController<
       if ('id' in blocked) {
         return (
           blocked.id === snapId &&
-          satisfiesVersionRange(snapInfo.version, blocked.versionRange)
+          satisfiesVersionRange(
+            snapInfo.version,
+            // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
+            blocked.versionRange as SemVerRange,
```

Comment:
`satisfiesVersionRange` and `blocked.versionRange` types seem to point to the same version of `@metamask/utils` for me locally. Is this type assertion needed?

Reply:
It is not - removed!